lib/mix/tasks/template_manager.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Thrifter.TemplateManager do
```

Comment:
This module doesn't match its file path and will probably break our Vim navigation.

Can we move this to `lib/thrifter/template_manager.ex` ?